components/teams-service/server/server.go

```diff
@@ -12,7 +12,6 @@ import (
 	"google.golang.org/grpc/status"
 
 	grpc_v2 "github.com/chef/automate/api/interservice/teams/v2"
```

Comment:
We cannot use just `grpc`, but should at least do this:
```suggestion
	grpc_iam "github.com/chef/automate/api/interservice/teams/v2"
```
And how much work to expunge the 'v2' on the end there...?

Reply:
I was going to leave that for this card: https://github.com/chef/automate/issues/3481